### PR TITLE
fix: update custom secret howto

### PR DESCRIPTION
**Pull request by vinckr**
- fix: update custom secrets howto
- chore: format

## Related Issue or Design Document

https://github.com/ory/kratos/discussions/2077

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: docs/helm/kratos.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ $ helm install \
 
 ## Configuration
```
Comment (Demonsthere) [inline docs/helm/kratos.md:34]:
```suggestion
## Kratos configuration
```

File: docs/helm/kratos.md [hunk 2]
```diff
@@ -55,48 +56,69 @@ $ helm install -f ./path/to/kratos-config.yaml ory/kratos
 
 Additionally, the following extra settings are available:
```
Comment (Demonsthere) [inline docs/helm/kratos.md:57]:
```suggestion
## Helm chart configuration
Additionally, the following extra settings are available:
```

Comment (Demonsthere) [reply docs/helm/kratos.md:57]: I think we could benefit from adding an extra header in the docs, and point to the full list of helm values and their description (so the readme of the chart)

Comment (vinckr) [reply docs/helm/kratos.md:61]: Done :), let me know what you think.

File: docs/helm/kratos.md [hunk 3]
```diff
@@ -53,50 +54,75 @@ and passing that as a value override to helm:
 $ helm install -f ./path/to/kratos-config.yaml ory/kratos
 ```
 
+## Helm Chart Configuration
+
+Have a look at the [Ory Kratos Helm Chart README](https://github.com/ory/k8s/tree/master/helm/charts/kratos) for the full list of Helm Chart values.
```
Comment (Demonsthere) [inline docs/helm/kratos.md:59]:
```suggestion
For the full list of chart values, please refer to the [Ory Kratos Helm Chart README](https://github.com/ory/k8s/tree/master/helm/charts/kratos), or the [values.yaml](https://github.com/ory/k8s/blob/master/helm/charts/kratos/values.yaml) directly 
```

**Review by Demonsthere — COMMENTED** [review]
Just one wording change ;)

**Review by Demonsthere — APPROVED** [review]

**Pull request merged**